clang/tools/libclang/CXDiagnosticSetDiagnosticConsumer.h

```diff
@@ -0,0 +1,41 @@
+//===- CXDiagnosticSetDiagnosticConsumer.cpp - CXDiagnosticSet producing
+// consumer -===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file provides a C API for extracting information from the clang driver.
```

Comment:
This comment looks like it's from another file. I would update it or even just remove it.